src/index.js

```diff
@@ -40,11 +54,16 @@ export default function loader(content) {
 
   let publicPath = null;
```

Comment:
`publicPath = `__webpack_public_path__ + ${JSON.stringify(outputPath)}`;` and remove else